docs/data-cookbooks/browse-generation.md

```diff
@@ -523,7 +528,7 @@ Their expected values from the example above may be useful in constructing a pro
 
 The payload includes a full list of files to be 'moved' into the cumulus archive.   The ```MoveGranules``` step will take this list, merge it with the information from input_granules adn move the files to their targets, then update the cmr metadata file if it exists with the updated granule locations.
```

Comment:
This section is outdated.

Reply:
I think this had been previously updated, I just missed the comment due to the lovely interface. 

I re-re-worded it as well, posting just this commit for brevity. 


d3bf4d6